metricbeat/module/haproxy/info/_meta/fields.yml

```diff
@@ -0,0 +1,196 @@
+- name: info
+  type: group
+  description: >
+    General infomration collected on HAProxy process
+  fields:
+    - name: nb_proc
+      type: integer
+      description: >
+        Number of processes
+
+    - name: process_num
+      type: integer
+      description: >
+        Process number
+
+    - name: pid
+      type: integer
+      description: >
+        Process ID
+
+    - name: uptime_sec
+      type: integer
+      description: >
+        Current uptime in seconds
+
+    - name: mem_max_mb
+      type: integer
+      format: bytes
```

Comment:
Oh, I see you do convert the value. So then I think the field just needs a rename to reflect the correct units.